free-programming-books-zh.md

```diff
@@ -639,6 +640,7 @@
 * [Python之旅](http://funhacks.net/explore-python) - Ethan
 * [Python教程 - 廖雪峰的官方网站](http://www.liaoxuefeng.com/wiki/0014316089557264a6b348958f449949df42a6d3a2e542c000)
 * [Tornado 6.1 中文文档](http://www.osgeo.cn/tornado/) - (网络上其他的都是较旧版本的，Online)
+* [人生苦短，我用python](https://www.cnblogs.com/derek1184405959/p/8579428.html) - (内含丰富的笔记以及各类教程)
```

Comment:
what's your opinion on the It's just a blog" issue?

Reply:
Even it is a blog, 
it has an index and enough content.
So I am ok to include this link